app/assets/javascripts/pageflow/editor/views/theme_item_view.js

```diff
@@ -0,0 +1,51 @@
+/** @api private */
+pageflow.ThemeItemView = Backbone.Marionette.ItemView.extend({
+  tagName: 'li',
+  template: 'pageflow/editor/templates/theme_item',
+  className: 'theme_item',
+
+  mixins: [pageflow.selectableView],
+
+  selectionAttribute: 'theme',
+
+  ui: {
+    themeName: '.theme_name',
+    useButton: '.use_theme',
+    inUseRegion: '.theme_in_use'
+  },
+
+  events: {
+    'click .use_theme': function(event) {
+      var newThemeName = this.model.get('name');
+
+      var stylesheetPath = this.options.themes.findByName(newThemeName)
+          .get('stylesheet_path');
+      pageflow.stylesheet.update('theme', stylesheetPath);
```

Comment:
This is the wrong place to handle stylesheet reloading since it couples the otherwise quite generic view to a very specific use case. There already is [an initializer concerned with reloading stylesheets](https://github.com/codevise/pageflow/blob/master/app/assets/javascripts/pageflow/editor/initializers/stylesheet_reloading.js). Let's move this functionality there to decoupled "how the theme is changed" (this view/ThemeInputView etc) from "what has to happen when the theme is changed" (mentioned initializer)